model/os/registry.yaml

```diff
@@ -75,6 +75,22 @@ groups:
         stability: development
         brief: 'Human readable operating system name.'
         examples: ['iOS', 'Android', 'Ubuntu']
+      - id: os.family
+        type: string[]
```

Comment:
Why an array? I would expect only one family per OS, so it would also have to have predefined values, as with `type`.